examples/example-next-13-next-auth/next.config.js

```diff
@@ -1,3 +1 @@
-module.exports = {
-  experimental: {appDir: true}
-};
+module.exports = {};
```

Comment:
Maybe we can remove the config files entirely now if they're empty?

Reply:
Good point, i didn't thought about it 😂